frontend/src/js/store/storehooks.ts

```diff
@@ -160,6 +161,7 @@ export const useAppInit = userId => {
     const multitenancy = hasMultitenancy || isHosted || isEnterprise;
     if (multitenancy) {
       tasks.push(dispatch(getUserOrganization()));
+      tasks.push(dispatch(getUserSubscription()));
     }
```

Comment:
since anything but hosted won't have the information accessible:
```suggestion
const multitenancy = hasMultitenancy || isHosted || isEnterprise;
if (multitenancy) {
tasks.push(dispatch(getUserOrganization()));
}
if (isHosted) {
      tasks.push(dispatch(getUserSubscription()));
}
```
but apart from this, I don't think we have to get this info here...